### ISSUE TITLE
Description set to build hint?

### ISSUE THREAD

**Issue by FriedrichFroebel** [hsnr-gamera/gamera-4#55]
When looking at the build/install log, I observed the following warning:

```python3
/opt/hostedtoolcache/Python/3.11.0-rc.2/x64/lib/python3.11/site-packages/setuptools/dist.py:160: UserWarning: newlines not allowed and will break in the future
  warnings.warn("newlines not allowed and will break in the future")
```

Skimming through the code, it seems like this is caused by the following lines, which define the `description` field value for the package: https://github.com/hsnr-gamera/gamera-4/blob/f37443466ba9aa78ef3a0a3896d92cd38fb9983c/setup.py#L231-L243

Is it correct that the package describes itself with some information regarding the build/installation requirements, instead of its actual purpose?

**Comment by FriedrichFroebel** [comment]
Will be resolved with #61 as well.

**Comment by cdalitz** [comment]
This code has been removed in the latest commit. The install option "-nowx" has been removed, and gamera will now install regardless whether wx is installed: if it is not present, the only effect will be that gamera_gui cannot be started.
